Mathlib/Algebra/Module/Zlattice.lean

```diff
@@ -222,6 +240,27 @@ theorem exist_unique_vadd_mem_fundamentalDomain [Finite ι] (x : E) :
   · exact (vadd_mem_fundamentalDomain b y x).mp h
 #align zspan.exist_unique_vadd_mem_fundamental_domain Zspan.exist_unique_vadd_mem_fundamentalDomain
 
+/-- The map `Zspan.fract_restrict` defines an equiv map between `E ⧸ span ℤ (Set.range b)`
+and `Zspan.fundamentalDomain b`. -/
+def QuotientEquiv [Fintype ι] :
+    E ⧸ span ℤ (Set.range b) ≃ (fundamentalDomain b) := by
+  refine Equiv.ofBijective ?_ ⟨fun x y => ?_, fun x => ?_⟩
+  · refine fun q => Quotient.liftOn q (fract_restrict b) (fun _ _ h => ?_)
+    rw [Subtype.mk.injEq, fract_restrict_apply, fract_restrict_apply, fract_eq_fract]
+    exact QuotientAddGroup.leftRel_apply.mp h
+  · refine Quotient.inductionOn₂ x y (fun _ _ hxy => ?_)
+    rw [Quotient.liftOn_mk (s := quotientRel (span ℤ (Set.range b))), fract_restrict,
+      Quotient.liftOn_mk (s := quotientRel (span ℤ (Set.range b))),  fract_restrict,
+      Subtype.mk.injEq] at hxy
+    apply Quotient.sound'
+    rwa [QuotientAddGroup.leftRel_apply, mem_toAddSubgroup, ← fract_eq_fract]
+  · obtain ⟨a, rfl⟩ := fract_restrict_surjective b x
+    exact ⟨Quotient.mk'' a, rfl⟩
+
+@[simp]
+theorem quotientEquiv_apply_mk [Fintype ι] (x : E) :
+    QuotientEquiv b (Submodule.Quotient.mk x) = fract_restrict b x := rfl
```

Comment:
Can you say anything about `QuotientEquiv.symm`?

Reply:
I added a new result
```lean
theorem quotientEquiv.symm_apply (x : fundamentalDomain b) :
    (QuotientEquiv b).symm x = Submodule.Quotient.mk ↑x 
```